### DISCUSSION TITLE
Different bar heights on laptop and external displays

### DISCUSSION THREAD

**Discussion by hamadahiro** [FelixKratz/SketchyBar#491] [Q&A]
I have my sketchybar set up to a height of 42, which perfectly covers the menubar, and vice versa. However, when I'm using my external display, the menubar only covers half of the sketchybar. Is there any workaround for this issue? Can I set two different heights for different displays? Please note that I always use only one display at a time.

**Comment by RaiSaugat** [comment]
@hamadahiro Are you using rectangle for window management or yabai?  I had the same issue when using rectangle, I switched to yabai and now it's working fine.

**Comment by hamadahiro** [comment]
I use yabai. Can you show your config files?

**Comment by Bain-OS** [comment]
Me too, did you find a solution?

**Comment by Bellavene** [comment]
You can script to change it.
The default commands to retrieve display info on macos are:

`system_profiler SPDisplaysDataType`

`defaults read /Library/Preferences/com.apple.windowserver.displays.plist`

**Reply by Bellavene** [reply]
Also, the difference that you have is related to DPI of your screen, check betterdisplay
